wondrous-app/components/Settings/Members/MembersTableRow/index.tsx

```diff
@@ -0,0 +1,128 @@
+import { useEffect, useState } from 'react';
+import Link from 'next/link';
+import MemberRoleDropdown from 'components/Settings/Members/MemberRoleDropdown';
+import { SafeImage } from 'components/Common/Image';
+import CopyIcon from 'components/Icons/copy';
+import { useWonderWeb3 } from 'services/web3';
+import PodIcon from 'components/Icons/podIcon';
+import { DropDown, DropDownItem } from 'components/Common/dropdown';
+import { TaskMenuIcon } from 'components/Icons/taskMenu';
+import MemberRoleSelectionDropdown from './MemberRoleSelectionDropdown';
+import {
+  DefaultProfilePicture,
+  UserOptions,
+  UserPodCount,
+  UserPodIconContainer,
+  UserProfile,
+  UserProfileDetailsContainer,
+  UserProfileName,
+  UserProfileNameContainer,
+  UserProfileUsername,
+  UserRowWrapper,
+  UserWalletAddress,
+  UserWalletAddressContainer,
+  WalletAddressEmptyState,
+} from './styles';
+import { addressTag } from './helpers';
+
+const MemberTableRow = ({ user, role, orgId, podId, roleList, promptRemoveUser }) => {
+  const [hasAddressBeenCopied, setHasAddressBeenCopied] = useState(false);
+  const [userWalletAddressTag, setUserWalletAddressTag] = useState('');
```

Comment:
introducing the concept of addressTag can be confusing, it's better if we just use associatedENS

Reply:
in that case, do you think `userENSNameOrWalletAddress` could be better ? since ENS is being given preference over wallet address and the state can store either ENS or wallet address.